payment/locale/fr/LC_MESSAGES/django.po

```diff
@@ -125,14 +125,14 @@ msgid ""
 msgstr ""
```

Comment:
Je m'aperçois que la formulation en anglais est très étrange : 

     "The proctored exam is organized by the provider in ProctorU only visible "
     "part of the course for students who have chosen the path checked"

Comment reformuler ?